### DISCUSSION TITLE
Keybindings of selection should be ignored while collection's searching bar is active

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#25766] [Q&A] [answered]
https://github.com/ppy/osu/assets/86934170/efb2c290-d733-46e6-bc33-9ebce52413f4

Even default `Space` binding.

**Comment by Theighlin** [comment]
I think something similar is happening with mod settings dropdowns, typing anything will select/deselect mods in the background

**Comment by Loreos7** [comment, marked as answer]
Fixed somewhere already
